### PR TITLE
support DAP subagency

### PR DESCRIPTION
**Pull request by ryanwoldatwork**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ github_info:
 
 # Configuration for DAP, add your agency ID here:
 # dap_agency: GSA
+# dap_subagency: TTS
```
Comment (afeld) [inline _config.yml:20]: Mind adding to the README as well?

**Review by afeld — CHANGES_REQUESTED** [review]
Nice, thanks!

File: _config.yml (hunk 1 above)
Comment (ryanwoldatwork) [reply _config.yml:20]: sure thing

Comment (ryanwoldatwork) [reply _config.yml:20]: updated ✅

**Comment by ryanwoldatwork** [comment]
This changeset may need to be ported to a previous release of uswds-jekyll - specifically because of older sites using an older version of uswds-jekyll, which in-turn uses an older version of USWDS (eg: 1.x).    

Thus, upgrading to latest uswds-jekyll would also update USWDS, which can break site layouts still on USWDS 1.x.

**Review by afeld — APPROVED** [review]
> This changeset may need to be ported to a previous release of uswds-jekyll

Sites using uswds-jekyll < 5 have bigger problems, unfortunately. Given there is zero dedicated team on this project, we don't have the ability to maintain multiple major versions.

**Pull request merged**